docs/development.md

```diff
@@ -10,7 +10,14 @@ This doc explains how to set up a development environment for Numaflow.
 1. [`protoc`](https://github.com/protocolbuffers/protobuf) for compiling protocol buffers
 1. [`Node.js®`](https://nodejs.org/en/) for running the UI
 1. [`yarn`](https://classic.yarnpkg.com/en/)
-1. [`k3d`](https://k3d.io/) for local development, if needed
+1. Local Kubernetes System - 
```

Comment:
Maybe not mentioning all the details is better.

```
1. A local Kubernetes cluster, [k3d](https://k3d.io/) or [minikube](https://minikube.sigs.k8s.io/docs/start/).
```

And not mentioning `docker` or `podman` at all might be better, because that is very tricky for the chosen dev cluster (`minikube` with `podman`? `k3d` with `docker` or `podman`?), that is something has to be figured out at the time installing the k8s cluster.